bpf/lbmap/lbmap.go

```diff
@@ -36,26 +41,82 @@ type ServiceKey interface {
 
 	// Returns a RevNatValue matching a ServiceKey
 	RevNatValue() RevNatValue
+
+	// Returns the port set in the key or 0
+	GetPort() uint16
+
+	// Set the backend index (master: 0, backend: nth backend)
+	SetBackend(int)
```

Comment:
I think it's cleaner to hide the actual size behind the API. We might support more than 64K backends at some point.